File: 02-Basic/python/06_decimal_to_binary.py
def decimal_to_binary(num):
    binary_list = []
    
    while num > 0:
        if num % 2 == 0:
            binary_list.append('0')
            num = num // 2
        else:
            binary_list.append('1')
            num = (num - 1) // 2
    
    binary_list.reverse()
    return ''.join(binary_list)

binary = decimal_to_binary(1505)
print(binary)

File: 02-Basic/JavaScript/06-decimalToBinary.js
 function decimalToBinary(num){
     list=[]
        while(num>0){
            
            if(num%2===0){
                num=num/2
                list.push("0")

            }
            else if(num%2!==0){
                list.push("1")
                num=num-1
                num=num/2
            }
            
        }
        list.reverse()
        return list.join("")
}



binary= decimalToBinary(1505)
console.log(binary)